### DISCUSSION TITLE
Burde vi ha "tabular numbers"-stil i tabellkolonner med talldata?

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#3368] [Annet]
_(Fake data, bare så det er sagt)_

https://user-images.githubusercontent.com/1223410/212857035-9058b6c1-b3d8-4b07-9d03-838083f1b5d6.mp4

Det slo meg her jeg satt med disse kolonnene at Saksnummer og Kundenummer så _uryddige_ ut. Det er en CSS-feature som heter [`font-variant-numeric`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric) hvor én av verdiene er `tabular-nums`. Den gjør i prinsippet tallene monospaced.

Jeg var ikke _helt_ fan av effekten på datoer, men synes Saksnummer og Kundenummer ble bedre. Er dette noe vi vil ha som en variant, og anbefale for tallkolonner?

**Comment by BjorneOma** [comment]
Jatakk! Massevis av tabeller i admin-løsningen vår som eg trur hadde sett bedre ut med dette.

**Comment by wkillerud** [comment]
Diskusjonen fortsetter i #3060
